cms/djangoapps/contentstore/views/entrance_exam.py

```diff
@@ -24,8 +24,8 @@
 from xmodule.modulestore.exceptions import ItemNotFoundError  # lint-amnesty, pylint: disable=wrong-import-order
 
 from ..helpers import remove_entrance_exam_graders
-from ..xblock_services.create_xblock import create_xblock
-from cms.djangoapps.contentstore.xblock_services.xblock_service import delete_item
+from ..xblock_storage_handlers.create_xblock import create_xblock
+from cms.djangoapps.contentstore.xblock_storage_handlers.view_handlers import delete_item
```

Comment:
Stylistic nit: Relative imports and absolute imports are both OK, but I think this would be easier to read if you chose one and used it consistently, eg:
```suggestion
from ..xblock_storage_handlers.create_xblock import create_xblock
from ..xblock_storage_handlers.view_handlers import delete_item
```